<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Demo</class>
 <widget class="QWidget" name="Demo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>680</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAddVert">
       <property name="text">
        <string>Добавить
вершину</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddEdge">
       <property name="text">
        <string>Добавить
ребро</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteVert">
       <property name="text">
        <string>Удалить 
вершину</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBSF">
       <property name="text">
        <string>Обход графа
в ширину</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblAnnotation">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="Graph" name="graph"/>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="btnNextStep">
       <property name="text">
        <string>Следующий 
шаг</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrevStep">
       <property name="text">
        <string>Предыдущий
шаг</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="tedAnnotation">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Graph</class>
   <extends>QGraphicsView</extends>
   <header>graph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
